### PR TITLE
Add SBI Sumishin Net Bank

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by phallobst — CHANGES_REQUESTED** [review]
We don't include the "www." prefix in the domain. If it's necessary, we use the `url` field.
```json
"domain": "netbk.co.jp",
"url": "https://www.netbk.co.jp",
```
This also requires the files to be moved/renamed, since they are named after the domain.

While we're at it, this SVG logo could be used instead of the PNG:

**Comment by ghost** [comment]
@phallobst Thank you very much. I fixed it.

**Comment by ghost** [comment]
Sorry, this is the first time I'm sending a pull request.
Is there anything else I should do?

File: entries/n/netbk.co.jp.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "SBI Sumishin Net Bank": {
+    "domain": "netbk.co.jp",
+    "url": "https://www.netbk.co.jp/",
+    "tfa": [
+      "custom-software"
+    ],
```
Comment (phallobst) [inline entries/n/netbk.co.jp.json:7]:
```suggestion
    ],
    "custom-software": [
      "Sumishin SBI Net Bank app"
    ],
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Once all requested changes are done, two reviewers will approve this PR and then it will be merged.

**Comment by ghost** [comment]
> Once all requested changes are done, two reviewers will approve this PR and then it will be merged.

Okay, I got it.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**